lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -79,14 +79,7 @@ def signal_access_token_required
       response.set_header(ACCESS_TOKEN_REQUIRED_HEADER, "true")
     end
 
-    def jwt_expire_at
-      expire_at = request.env["jwt.expire_at"]
-      return unless expire_at
-
-      expire_at - 5.seconds # 5s gap to start fetching new token in advance
-    end
```

Comment:
Extracted these 'jwt' helpers to be inside `WithShopifyIdToken` concern for reusability